controllers/apim/ratelimitpolicy_cluster_envoy_filter.go

```diff
@@ -70,21 +70,6 @@ func (r *RateLimitPolicyReconciler) gatewayRateLimitingClusterEnvoyFilter(
 	gwKey := client.ObjectKeyFromObject(gw)
 	logger.V(1).Info("gatewayRateLimitingClusterEnvoyFilter", "gwKey", gwKey, "rlpRefs", rlpRefs)
 
-	// Load all relevant rate limit policies
```

Comment:
this changes has nothing to do with the purpose of the PR. This code was just useless.